src/EnergyPlus/PluginManager.cc

```diff
@@ -402,7 +403,7 @@ namespace PluginManagement {
 #endif
     }
 
-    void registerNewCallback(int iCalledFrom, const std::function<void()> &f)
+    void registerNewCallback(EnergyPlusData &EP_UNUSED(state), int iCalledFrom, const std::function<void(void *)> &f)
```

Comment:
Right now the callbacks are stored in an EnergyPlus "global" variable, so this state isn't necessary at the moment.  However, at a later date, the callbacks will be stored in this "state" structure, so I just added this arg now to make future changes easier.